tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/GenericAuthorityTests.cs

```diff
@@ -21,28 +22,37 @@ namespace Microsoft.Identity.Test.Unit.CoreTests.InstanceTests
     public class GenericAuthorityTests : TestBase
     {
         [DataTestMethod]
```

Comment:
You haven't implemented / tested the fact that the `ExtraParamsToCache` should appear in the `AuthenticationResult.AdditionalData`